NFIQ2/NFIQ2Algorithm/src/tool/nfiq2_ui_refresh.cpp

```diff
@@ -50,6 +50,119 @@ namespace BE = BiometricEvaluation;
 
 std::mutex mutGray;
 
+// Wrappers for yesOrNo Prompts
+bool
+NFIQ2UI::askIfQuantize()
+{
+	const std::string prompt =
+	    "This Image is not 8 bit or 1 bit grayscale. Would you like to "
+	    "quantize/convert this image to 8 bit color and depth?";
+	return NFIQ2UI::yesOrNo(prompt, false, true, true);
+}
+
+bool
+NFIQ2UI::askIfDefaultResolution(const std::string &name,
+    const uint16_t defaultDPI, const uint16_t requiredDPI)
+{
+	// 72PPI is default resolution
+	if (defaultDPI != 72) {
+		return false;
+	}
```

Comment:
If you're passing this is a parameter, and it has to be 72, then don't ask for it as a parameter?